### ISSUE TITLE
Media buttons should have symbols, with alt text. Not regular text.

### ISSUE THREAD

**Issue by Myk-edu** [Myk-edu/adv-prog-project-ocr#5]
Change labels, add alt.

**Comment by Myk-edu** [comment]
Use e.g.  QPushButton.setAccessibleName("play")
